### PR TITLE
Let candidates proceed if they decide not to add another job

### PR DESCRIPTION
**Pull request by TobyRet**
## Context

Currently, if you add a job, choose to add another, but then change your mind, options for exiting out of this state are non-optimal.

## Changes proposed in this pull request

- Redirect to work history review page if candidate submits blank form
- ‘Add another job’ radio now defaults to false
- Horizontal line removed in new job form
- Adding another qualification radio button defaults to 'no'

## Guidance to review

- Create a job then choose to add another. When you submit the form (keeping all fields blank) you should be redirected back to the Work Experience review page.
- When adding another ‘Academic and other relevant qualification’ the ‘Do you want to add another qualification?’ radio button selection should now defaults to 'No, not at the moment' 

## Link to Trello card

https://trello.com/c/wbUacSal/1974-let-candidates-proceed-if-decide-not-to-add-another-qualification-or-job

## Things to check

- [x] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] API release notes have been updated if necessary
- [ ] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

**Comment by paulrobertlloyd** [comment]
This looks good to me! I so slightly wonder if we need to have both fixes at the same time, but not sure how closely we are monitoring this to be able to make a decision on whether one or both are needed (I’m not entirely sure about defaulting to ‘No’, for example).

**Review by davidgisbey — APPROVED** [review]
Looks good. I tested it's working nicely. Not related to this, but this will get more complicated on other quals if we try and implement something similar as we prefill values in some contexts.

**Review by paulrobertlloyd — APPROVED** [review]

File: app/controllers/candidate_interface/work_history/edit_controller.rb [hunk 1]
```diff
@@ -14,18 +14,20 @@ def new
                                   start_date_year: start_date.year,
                                   end_date_month: end_date&.month || '',
                                   end_date_year: end_date&.year || '',
-                                  add_another_job: true,
+                                  do_not_add_another_job: true,
```
Comment (frankieroberto) [inline app/controllers/candidate_interface/work_history/edit_controller.rb:17]: Any reason not to use `add_another_job: false`?

File: app/forms/candidate_interface/other_qualification_wizard.rb [hunk 2]
```diff
@@ -0,0 +1,226 @@
+module CandidateInterface
+  class OtherQualificationWizard
```
Comment (frankieroberto) [inline app/forms/candidate_interface/other_qualification_wizard.rb:2]: I can't see whether this new class is referenced from at all? 🤔

File: app/controllers/candidate_interface/work_history/edit_controller.rb (hunk 1 above)
Comment (TobyRet) [reply app/controllers/candidate_interface/work_history/edit_controller.rb:17]: Yeah that would be ideal, however we now need to do reverse the logic in order to pre-select the correct radio button.
See https://github.com/DFE-Digital/apply-for-teacher-training/pull/3478

Comment (paulrobertlloyd) [reply app/controllers/candidate_interface/work_history/edit_controller.rb:17]: Not sure I follow… `add_another_job: false` seems much clearer. Can we not reverse the boolean on the form?

Comment (TobyRet) [reply app/controllers/candidate_interface/work_history/edit_controller.rb:17]: @paulrobertlloyd The radio button needs to resolve to `true` in order to be selected. In response to the question 'Do you want to add another job?' we want 'No' to be selected, not 'Yes' right?

File: app/forms/candidate_interface/other_qualification_wizard.rb (hunk 2 above)
Comment (TobyRet) [reply app/forms/candidate_interface/other_qualification_wizard.rb]: Sorry - this was the result of a bad rebase. Steve had done some refactoring in another PR. Fixed now.

File: app/controllers/candidate_interface/work_history/edit_controller.rb (hunk 1 above)
Comment (frankieroberto) [reply app/controllers/candidate_interface/work_history/edit_controller.rb:17]: @TobyRet can't we just initialise the form with the `no` option selected by default, via `WorkExperienceForm.new(add_another_job: false)`?

Comment (TobyRet) [reply app/controllers/candidate_interface/work_history/edit_controller.rb]: 👍 Yep no problem @frankieroberto. I've made the changes

**Pull request merged**